src/sentry/integrations/jira/notify_action.py

```diff
@@ -160,15 +161,16 @@ def after(self, event, state):
         def create_issue(event, futures):
             """Create the Jira ticket for a given event"""
 
-            # TODO check if a Jira ticket already exists for the given event's issue. if it does, skip creating it
-            resp = installation.create_issue(self.data)
-            ExternalIssue.objects.create(
-                organization_id=organization.id,
-                integration_id=integration.id,
-                key=resp["key"],
-                title=event.title,
-                description=installation.get_group_description(event.group, event),
-            )
+            # HACK to get fixVersion in the correct format
+            if not isinstance(self.data["fixVersions"], list):
+                self.data["fixVersions"] = [self.data["fixVersions"]]
+
+            if self.data.get("dynamic_form_fields"):
+                del self.data["dynamic_form_fields"]
+
+            if not self.has_linked_issue(event, integration):
```

Comment:
Can we add an `else` where we log something like "rule triggered for existing ticket {}"?

Reply:
@mgaeta sure, do you want to log when one was created too?

Reply:
I'm not sure. We should already know when they're created in the DB, but I don't think more logging is bad.